opendebates/views.py

```diff
@@ -212,9 +212,28 @@ def vote(request, id):
             'form': form,
             'idea': idea,
             # 'comment_form': CommentForm(idea),
-            }
+        }
     state = state_from_zip(form.cleaned_data['zipcode'])
 
+    session_key = request.session.session_key or ''
```

Comment:
It can happen in tests, if one isn't careful about it.  I don't _think_ it can happen in real life, but felt it was best to proceed as if it could.